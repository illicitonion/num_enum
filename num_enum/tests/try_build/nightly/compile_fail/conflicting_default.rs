#![feature(derive_default_enum)]

#[derive(Default, num_enum::Default)]
#[repr(u8)]
enum Number {
    #[default]
    Zero,
}

fn main() {
}
