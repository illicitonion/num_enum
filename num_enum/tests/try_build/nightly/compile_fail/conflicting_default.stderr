error[E0119]: conflicting implementations of trait `std::default::Default` for type `Number`
 --> $DIR/conflicting_default.rs:3:19
  |
3 | #[derive(Default, num_enum::Default)]
  |          -------  ^^^^^^^^^^^^^^^^^ conflicting implementation for `Number`
  |          |
  |          first implementation here
  |
  = note: this error originates in the derive macro `num_enum::Default` (in Nightly builds, run with -Z macro-backtrace for more info)
